packages/hadron-app-registry/src/app-registry.ts

```diff
@@ -161,13 +154,8 @@ export class AppRegistry {
    *
    * @returns {Store} The store.
    */
-  getStore(name: string): Store {
-    const store = this.stores[name];
-    if (store === undefined) {
-      this.storeMisses[name] = (this.storeMisses[name] || 0) + 1;
-      return STUB_STORE;
-    }
```

Comment:
As far as I can tell nothing depended on this behavior and this actually allows us to implement "get or create" logic for roles inside collection tab